### ISSUE TITLE
Bytecode changes for dict in python 3.5 causes batavia.core.Dict to fail

### ISSUE THREAD

**Issue by splab** [pybee/batavia#26]
With python 3.5 the bytecode for storing data to the dict has changed to:

```
  2           0 LOAD_CONST               1 ('a')
              3 LOAD_CONST               2 (1)
              6 LOAD_CONST               3 ('b')
              9 LOAD_CONST               4 (2)
             12 BUILD_MAP                2
             15 STORE_FAST               0 (lonk)
```

causing dicts to be empty.

To reproduce use (in the testbed):

``` javascript
foo = {'a': 1, 'b': 2}
print('bar', foo)
```

Expected result: bar 1
Actual result: bar

**Comment by miki725** [comment]
so the fix is available on 3.5 branch. will try to get it running for both 3.4 and 3.5

**Comment by slick666** [comment]
This issue for 3.5 has been addresses. issue in Python 3.6 are continues in #181

**Comment by judy2k** [comment]
@freakboy3742 I think this issue has been accidentally re-opened. There's a separate issue for Python 3.6: #181

**Comment by Zac-HD** [comment]
Was this fixed by #89?

**Comment by freakboy3742** [comment]
Yup!
